### PR TITLE
chore(sdk): Update passkeys documentation

### PR DESCRIPTION
**Pull request by yagopv**
We are updating the passkeys guide and protocol-kit reference section after implementing compatibility with React Native

### CODE REVIEW

File: pages/sdk/signers/passkeys.mdx [hunk 1]
```diff
@@ -98,44 +100,68 @@ In this guide, you will learn how to create a Passkey signer that can be added a
...
+  const protocolKit = await Safe.init({ provider, signer, safeAddress })
+  
+  const transaction = { to: '0x1234', value: '0x0', data: '0x' }
+  const safeTransaction = await protocolKit.createTransaction({ transactions: [transaction] })
+  const signedSafeTransaction = await protocolKit.signTransaction(safeTransaction)
+  ```
+  
+  {/* <!-- vale on --> */}
+  
+  </Steps>
```
Comment (germartinez) [inline pages/sdk/signers/passkeys.mdx:164]:
```suggestion
</Steps>
```

File: pages/sdk/signers/passkeys.mdx [hunk 2]
```diff
@@ -98,44 +100,68 @@ In this guide, you will learn how to create a Passkey signer that can be added a
...
 
   {/* <!-- vale on --> */}
 
-  At this point, it's critical to securely store the information in the `passkey` object in a persistent service. Losing access to this data will result in the user being unable to access their passkey and, therefore, their Safe Smart Account.
+  At this point, it's critical to securely store the information in the `passkeySigner` object in a persistent service. Losing access to this data will result in the user being unable to access their passkey and, therefore, their Safe Smart Account.
 
   ### Get the provider and signer
 
-  Once the passkey is created, you can get the `provider` and `signer`, which is the externally-owned account of the user that was derived from its credentials.
+  Once the passkey is created, you need the `provider` and `signer` props that the Safe\{Core\} SDK require to instantiate the kits.
```
Comment (germartinez) [inline pages/sdk/signers/passkeys.mdx:112]:
```suggestion
  Once the passkey is created, you need the `provider` and `signer` properties required to instantiate the Safe\{Core\} SDK kits.
```

File: pages/sdk/signers/passkeys.mdx [hunk 3]
```diff
@@ -98,44 +100,68 @@ In this guide, you will learn how to create a Passkey signer that can be added a
...
+    const signer = passkeySigner
       ```
+          </Tabs.Tab>
+    <Tabs.Tab>
+    ```typescript
+    import { sepolia } from 'viem/chains'
+
+    const provider = sepolia.rpcUrls.default.http[0]
+    const signer = passkeySigner
+    ```
```
Comment (germartinez) [inline pages/sdk/signers/passkeys.mdx:140]: I think this is out of scope for this PR.
The used `provider` is just an example to provide to the kits, all the different ways to create it are covered in the reference here: https://docs.safe.global/reference-sdk-protocol-kit/initialization/init#provider

In fact, I think the `provider` could also be reemoved as this is just about the generating the `signer`. This also applies to all the Safe Signers guides, so I think it's out of scope for this PR.

**Review by germartinez — CHANGES_REQUESTED** [review]

File: pages/sdk/protocol-kit/reference/safe.mdx [hunk 4]
```diff
@@ -1081,3 +1081,32 @@ const signedMessage = await protocolKit.signMessage(signedMessage, SigningMethod
 const signedMessage = await protocolKit.signMessage(signedMessage, SigningMethod.ETH_SIGN)
 const signedMessage = await protocolKit.signMessage(signedMessage, SigningMethod.SAFE_SIGNATURE, parentSafeAddress).
 ```
+
```
[the remark this answers is not in the document]
Comment (germartinez) [reply pages/sdk/protocol-kit/reference/safe.mdx:1084]: I just rebased the `developement` branch with the latest changes from `main`

File: pages/sdk/signers/passkeys.mdx (hunk 3 above)
Comment (yagopv) [reply pages/sdk/signers/passkeys.mdx:140]: What is ooo? the two examples or the next section "Instantiate the SDK"?

Comment (germartinez) [reply pages/sdk/signers/passkeys.mdx:140]: The changes inside ```<Tabs items={['eip1193', 'rpcUrl']}>```

Comment (yagopv) [reply pages/sdk/signers/passkeys.mdx]: done

File: pages/sdk/signers/passkeys.mdx [hunk 5]
```diff
@@ -98,43 +100,49 @@ In this guide, you will learn how to create a Passkey signer that can be added a
...
+  You can instantiate the provider using `viem` and the following imports:
 
-      ```typescript
-      import { createWalletClient, http } from 'viem'
-      import { sepolia } from 'viem/chains'
-      ```
+  ```typescript
+  import { createWalletClient, http } from 'viem'
+  import { sepolia } from 'viem/chains
+  const provider = createWalletClient({
```
Comment (germartinez) [inline pages/sdk/signers/passkeys.mdx:121]:
```suggestion
  import { sepolia } from 'viem/chains
  
  const provider = createWalletClient({
```

File: pages/sdk/signers/passkeys.mdx [hunk 6]
```diff
@@ -89,7 +91,7 @@ In this guide, you will learn how to create a Passkey signer that can be added a
 
   {/* <!-- vale on --> */}
 
-  After generating the `passkeyCredential` object, you need to create a new object with the `PasskeyArgType` type that will contain the `rawId` and the `coordinates` information.
+  After generating the `passkeyCredential` object, you need to create the signer. This signer will be an `PasskeyArgType` object containing the `rawId` and the `coordinates` information.
```
Comment (germartinez) [inline pages/sdk/signers/passkeys.mdx:94]:
```suggestion
  After generating the `passkeyCredential` object, you need to create the signer. This signer will be a `PasskeyArgType` object containing the `rawId` and the `coordinates` information.
```

File: pages/sdk/signers/passkeys.mdx [hunk 7]
```diff
@@ -98,43 +100,49 @@ In this guide, you will learn how to create a Passkey signer that can be added a
...
-  At this point, it's critical to securely store the information in the `passkey` object in a persistent service. Losing access to this data will result in the user being unable to access their passkey and, therefore, their Safe Smart Account.
+  At this point, it's critical to securely store the information in the `passkeySigner` object in a persistent service. Losing access to this data will result in the user being unable to access their passkey and, therefore, their Safe Smart Account.
 
   ### Get the provider and signer
 
-  Once the passkey is created, you can get the `provider` and `signer`, which is the externally-owned account of the user that was derived from its credentials.
+  Once the passkey is created, you need the `provider` and `signer` properties required to instantiate the Safe\{Core\} SDK kits.
 
-  {/* <!-- vale off --> */}
+  [How to initialize the Protocol Kit](https://docs.safe.global/reference-sdk-protocol-kit/initialization/init)
```
Comment (germartinez) [inline pages/sdk/signers/passkeys.mdx:114]:
```suggestion
Check [how to initialize the Protocol Kit](../../reference-sdk-protocol-kit/initialization/init.mdx)
```

File: pages/sdk/signers/passkeys.mdx [hunk 8]
```diff
@@ -98,43 +100,49 @@ In this guide, you will learn how to create a Passkey signer that can be added a
...
 
-  <Tabs items={['viem']}>
-    <Tabs.Tab>
-      You can instantiate the provider using `viem` and the following imports:
+  You can instantiate the provider using `viem` and the following imports:
```
Comment (germartinez) [inline pages/sdk/signers/passkeys.mdx:116]: Can you add the <Tabs/> component here like before?

File: yarn.lock [hunk 9]
```diff
@@ -0,0 +1,9639 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (germartinez) [inline yarn.lock:1]: `yarn` is not used in the docs repo, but `pnpm` instead. `yarn.lock` file should be removed.

Comment (yagopv) [reply yarn.lock:1]: ah yeah, sorry 👍🏼

File: pages/sdk/signers/passkeys.mdx [hunk 10]
```diff
@@ -98,16 +100,18 @@ In this guide, you will learn how to create a Passkey signer that can be added a
...
 
-  At this point, it's critical to securely store the information in the `passkey` object in a persistent service. Losing access to this data will result in the user being unable to access their passkey and, therefore, their Safe Smart Account.
+  At this point, it's critical to securely store the information in the `passkeySigner` object in a persistent service. Losing access to this data will result in the user being unable to access their passkey and, therefore, their Safe Smart Account.
 
   ### Get the provider and signer
 
-  Once the passkey is created, you can get the `provider` and `signer`, which is the externally-owned account of the user that was derived from its credentials.
+  Once the passkey is created, you need the `provider` and `signer` properties required to instantiate the Safe\{Core\} SDK kits.
+
+Check [how to initialize the Protocol Kit](../../reference-sdk-protocol-kit/initialization/init.mdx)
```
Comment (germartinez) [inline pages/sdk/signers/passkeys.mdx:114]:
```suggestion
  Check [how to initialize the Protocol Kit](../../reference-sdk-protocol-kit/initialization/init.mdx)
```

**Review by germartinez — APPROVED** [review]

**Pull request merged**